src/classes/vscode-launch.ts

```diff
@@ -17,20 +17,7 @@
 
 import { TDMLConfig } from './tdmlConfig'
 import { DataEditorConfig } from './dataEditor'
-
-export interface InfosetOutput {
-  type: string
-  file: string
-}
-
-export interface LoggingConfig {
-  level: string
-  file: string
-}
```

Comment:
I think the `LoggingConfig` interface is generic enough and should stay here since it can be reused by the other components, like the Data Editor.